app/code/Magento/AdvancedSearch/Test/Mftf/Section/AdminSearchTermsPageGridActionsSection.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<sections xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Page/etc/SectionObject.xsd">
+    <section name="AdminSearchTermsPageGridActionsSection">
+        <element name="addNewSearchTerm" type="button" selector="#add" timeout="30"/>
```

Comment:
Please, feel free to reuse the [existing](https://github.com/magento/magento2/blob/dfd785f5f2934a4c79262d44c4ff5543ba577b51/app/code/Magento/Backend/Test/Mftf/Section/AdminMainActionsSection.xml#L15) element

Reply:
@rogyar I will use the existing action groups. This file is not used in anywhere now. So removing this file.